e2e_samples/unstructured_data/src/evaluators/exact_numeric_match.py

```diff
@@ -42,12 +42,11 @@ class NumericEvaluator:
             - `1,234.56` (without dollar sign)
     """
 
-    def __init__(self, truth_key: str, **kwargs: Any) -> None:
-        self.truth_key = truth_key
```

Comment:
truth_key is no longer needed. The experiment orchestrator outputs as a flat list so that we can easily map the columns when passing to the evaluate function. Ex: {"truth": {"total-revevue": "some-value"}}` -> {"truth.total-revenue": "some-value"}